backend/src/test/java/com/team4/backend/testdata/StudentMockData.java

```diff
@@ -38,7 +39,10 @@ public static StudentCreationDto getMockStudentDto() {
     }
 
     public static Set<String> getListExclusiveOfferIds() {
```

Comment:
If you put a set for the exclusiveOffersId, then why you don't put appliedOffersId as a Set in Student entity?
Is it necessary to put a Set, if yes, you should put it for every list that we put on entity. Personally i don't think is necessary, I think it is a good idea, but we are already checking with .contains(), we should follow what we started with or else is just better to all change it to a Set<> collection, i'm waiting for your feedback on this